src/core_functions/aggregate/holistic/quantile.cpp

```diff
@@ -288,6 +288,7 @@ struct QuantileListOperation : QuantileOperation {
 
 		auto v_t = state.v.data();
 		D_ASSERT(v_t);
+		(void)v_t;
```

Comment:
I think `D_ASSERT(state.v.data());` would avoid the variable?

Reply:
Sure, that would work, but we need the assert in place, or we should just remove all mentions of `state.v.data()`?

Reply:
Not sure what you mean - the assert is just that the memory has been allocated. If not, there is a serious problem...